app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -644,7 +647,7 @@ def alert_get_info(alert)
       @alert_profiles = @alert.memberof.sort_by { |p| p.description.downcase }
     end
 
-    unless @alert.expression.kind_of?(MiqExpression) # Get the EMS if it's in the expression
+    if @alert.expression && !@alert.expression.kind_of?(MiqExpression) # Get the EMS if it's in the expression
       @ems = ExtManagementSystem.find_by_id(@alert.expression[:options][:ems_id].to_i)
```

Comment:
Also note that this commit only adds additional check for @alert.expression rest of the logic comes from the original work.